packages/babel-plugin/src/utils/css-builders.ts

```diff
@@ -488,6 +489,16 @@ const extractObjectExpression = (node: t.ObjectExpression, meta: Metadata): CSSO
         return;
       }
 
+      if (isEmptyValue(propValue)) {
+        css.push({
+          type: 'unconditional',
+          // Let postCSS strip out later
+          css: `${kebabCase(key)}: undefined;`,
```

Comment:
I'm a bit concerned by this as it is technically invalid CSS. That adds a lot of potential for bugs if we don't handle all of these cases safely.

Could you avoid pushing the CSS entirely in this case?

Reply:
Ah dang that was really silly of me!  I can just return here. Nice catch!